docs/source/08_logging/01_logging.md

```diff
@@ -18,6 +18,8 @@ log.warning("Issue warning")
 log.info("Send information")
 ```
 
+Note: Log names like the example above, `__name__`, can be handler names defined within the `handlers` section keys.  The default handler names are `console, info_file_handler, error_file_handler` found within `conf/base/logging.yml`.
```

Comment:
````suggestion
```eval_rst
.. note:: The name of a logger corresponds to the keys of the ``loggers``  sections in ``logging.yml`` (e.g. ``kedro.io``). See `Python's logging documentation <https://docs.python.org/3/library/logging.html#logger-objects>`_ for more information.
```
````